security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -77,6 +78,11 @@ type sdsConnection struct {
 
 	// The secret associated with the proxy.
 	secret *model.SecretItem
+
+	// Full resource Name from coming discovery request.
```

Comment:
Can you define these into some constants instead of relying on hard-coding and comments?

Reply:
s/coming/incoming/
?

Reply:
updated comments, the fullresourcename is constructed in pilot original(and put const there).